{{ cookiecutter.github_repo_name }}/tests/test_calculator.py

```diff
@@ -1,17 +1,35 @@
+import pytest
 from {{cookiecutter.package_dir_name}} import calculator
 
 
 def test_dot_product_2D():
-    """Test the dot product function in 2D."""
+    """Test the dot product function with 2D vectors."""
     a = [1, 2]
     b = [3, 4]
     expected = 11
-    assert calculator.dot_product(a, b) == expected
+    actual = calculator.dot_product(a, b)
+    assert actual == expected
 
 
 def test_dot_product_3D():
-    """Test the dot product function in 3D."""
+    """Test the dot product function with 3D vectors."""
     a = [1, 2, 3]
     b = [4, 5, 6]
     expected = 32
-    assert calculator.dot_product(a, b) == expected
+    actual = calculator.dot_product(a, b)
+    assert actual == expected
+
+
+@pytest.mark.parametrize(
```

Comment:
Also providing another intuitive example on how to use `parametrize` suggested by @sbillinge - also follow the group's best practices provided in the official doc.